### PR TITLE
Add application launcher to masthead

### PR DESCRIPTION
**Pull request by rhamilto**
screen shot 2019-01-10 at 2 56 50 pm
Note there is a divider after "Cluster Manager", but it blends in with the active state on the first item (which is PatternFly default behavior).  PatternFly should probably consider making the divider and the active state different colors?

screen shot 2019-01-10 at 11 26 44 am

TODOs:

- [x] determine what the correct link label and link are [@serenamarie125]
- [x] determine mobile behavior--does the link move to the kebab? [@serenamarie125]
- [x] only show application launcher based on some yet-to-be determined flag

EXISTING KNOWN ISSUES:
* usernames do not truncate, so very long usernames can break the layout.  I will address in a separate PR as the upstream fix has not yet landed.

### CODE REVIEW

**Comment by serenamarie125** [comment]
Per Mobile behavior, yes they should be in the kebab.  We are following up with the PF4 team on  3 main questions (1) what's the order (2) is it in a flat hierarchy (3) should there be separators between "sections"

Right now it looks like things are added right to left - which seems opposite from what I'd expect.  Either @beanh66 or I will come back with what we find out.

**Comment by serenamarie125** [comment]
>  determine what the correct link label and link are [@serenamarie125]

@rhamilto 
label is _Cluster Manager_
link is _cloud.openshift.com_

**Comment by alimobrem** [comment]
Can we add in a placeholder link for Service Mesh in to the launcher?

**Comment by serenamarie125** [comment]
@rhamilto, at mobile, the desired kebab menu contents is as follows:

Cluster Manager
---
Documentation
About
----
Log out

**Comment by rhamilto** [comment]
> Can we add in a placeholder link for Service Mesh in to the launcher?

@spadgett and I discussed this and are of the opinion we can do that later as a bug fix.  The important thing now is to get the feature in.

File: frontend/public/components/masthead-toolbar.jsx [hunk 1]
```diff
@@ -106,6 +127,19 @@ class MastheadToolbar_ extends React.Component {
...
 
+  _renderMenuItems(actions) {
+    return actions.map((action, i) => {
+      if (action.separator) {
+        return <DropdownSeparator key={i} />;
+      }
+      return (
+        <DropdownItem key={i} onClick={action.callback}>
+          {action.label}
+        </DropdownItem>
```
Comment (spadgett) [inline frontend/public/components/masthead-toolbar.jsx]: nit better as a ternary

```js
return action.separator
  ? <DropdownSeparator key={i} />
  : <DropdownItem key={i} onClick={action.callback}>{action.label}</DropdownItem>;

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**